mpp/build.gradle.kts

```diff
@@ -58,8 +58,7 @@ val mainComponents =
         ComposeComponent(":navigation:navigation-common", viewModelPlatforms),
         ComposeComponent(":navigation:navigation-runtime", viewModelPlatforms),
 
-        //To be added later: (also don't forget to add gradle.properties see in lifecycle-runtime for an example)
-        ComposeComponent(":lifecycle:lifecycle-runtime-compose"),
+        ComposeComponent(":lifecycle:lifecycle-runtime-compose", supportedPlatforms = ComposePlatforms.ALL),
```

Comment:
Is it needed? I don't see it in other `lifecycle`'s

Reply:
<img width="951" alt="Screenshot 2024-04-26 at 11 33 20" src="https://github.com/JetBrains/compose-multiplatform-core/assets/1836384/54d3d7dc-96af-479f-93cd-ae508f898845">

Reply:
`ALL` instead of `ALL_AOSP` because of `androidXComposeMultiplatform` plugin in this module